### PR TITLE
LOOK-244 fix for warehouse_order_items check_table

### PR DESCRIPTION
**Pull request by Simarglok**

### CODE REVIEW

File: update_everything_mapping_test.py [hunk 1]
```diff
@@ -100,6 +100,36 @@ def success_slack_alert(context):
     return alert.execute(context=context)
 
 
+def check_table_woi(job_name, schema):
+    target_schema = schema
+    table = job_name
+    source_fields = 'id, order_id, product_id, "name", descr, price_type, price_per, charge_by, price, qty, qty_free, amount, tax, discount_value, discount_type_bak, total_amount, created_at, updated_at, is_charge_by_order, is_free, free_discount, income, discount_amount, item_type, count, special_id, special_item_id, is_half_eighth, is_returned, returned_amount, discount_type, free_amount, paid_amount, wcii_cart_item, sync_created_at, sync_updated_at, product_checkin_id, is_excise, returned_at, is_marijuana_product, product_is_tax_exempt, is_metrc, is_under_package_control, base_amount, discount_id, delivery_tax, discount_count, is_exchanged, exchanged_at, product_brutto_weight, product_brutto_weight_validation, confirmed_at'
```
Comment (iplotnikov-telemedicine) [inline update_everything_mapping_test.py:106]: @Simarglok could you please declare it as a list and below in the query unpack the list into a string? It seems to be a more 'pythonic' way

File: update_everything_mapping.py [hunk 2]
```diff
@@ -101,6 +101,37 @@ def success_slack_alert(context):
     return alert.execute(context=context)
 
 
+def check_table_woi(job_name, schema):
+    target_schema = schema
+    table = job_name
+    source_fields = 'id, order_id, product_id, "name", descr, price_type, price_per, charge_by, price, qty, qty_free, amount, tax, discount_value, discount_type_bak, total_amount, created_at, updated_at, is_charge_by_order, is_free, free_discount, income, discount_amount, item_type, count, special_id, special_item_id, is_half_eighth, is_returned, returned_amount, discount_type, free_amount, paid_amount, wcii_cart_item, sync_created_at, sync_updated_at, product_checkin_id, is_excise, returned_at, is_marijuana_product, product_is_tax_exempt, is_metrc, is_under_package_control, base_amount, discount_id, delivery_tax, discount_count, is_exchanged, exchanged_at, product_brutto_weight, product_brutto_weight_validation, confirmed_at'
```
Comment (iplotnikov-telemedicine) [inline update_everything_mapping.py:107]: @Simarglok please see the comment below

**Review by iplotnikov-telemedicine — APPROVED** [review]

File: update_everything_mapping_test.py (hunk 1 above)
Comment (Simarglok) [reply update_everything_mapping_test.py]: move this to yaml file

File: update_everything_mapping.py (hunk 2 above)
Comment (Simarglok) [reply update_everything_mapping.py]: move this to yaml file

**Review by denis-prk — APPROVED** [review]

File: python/core/connections.py [hunk 3]
```diff
@@ -12,12 +15,30 @@ def redshint_conn_dev():
     return redshift_conn
 
 
-# Get connection to Redshift DB on production environment
+# Get connection to Redshift DB on dev environment
 def redshint_conn_mock():
```
[the remark this answers is not in the document]
Comment (Simarglok) [reply python/core/connections.py]: fixed

**Pull request merged**